### ISSUE TITLE
Bazel error: v3.21.12 archive checksum changed

### ISSUE THREAD

**Issue by johanbrandhorst** [protocolbuffers/protobuf#11354]
**What version of protobuf and what language are you using?**

Version: v3.21.12
Language: N/A

**What operating system (Linux, Windows, ...) and version?**

Linux 

**What did you do?**

I'm the maintainer of the grpc-gateway. We use MEND Renovate to keep our Bazel protoc dependency up to date. It updated to v3.21.12 in the 13th of December in https://github.com/grpc-ecosystem/grpc-gateway/pull/3068. It seems that since then, the checksum for the archive served on this tag has changed, which is causing errors in our CI jobs (https://github.com/grpc-ecosystem/grpc-gateway/actions/runs/3720083266/jobs/6309341191).

Please advise whether the checksum did indeed change and it is safe to update the stored checksum.

**Comment by zhangskz** [comment]
tldr: Checksum is safe to update, and indeed changed

It seems during our release process, we repushed the 3.21.12 tag to a slightly different commit. The new and old commit should be functionally the same, but potentially at different commits from before/after we updated the CHANGES.txt file with our changelog. This step in our release should now be obsolete, and this should not happen again.

Note, you should probably be using v21.12 (not v3.21.12) which points to the same release commit. The per-language tag is only intended for objective-c and c++ specific releases and may be removed in the future.
